dao/src/test/java/se/fortnox/reactivewizard/db/DaoTransactionsTest.java

```diff
@@ -60,78 +54,84 @@ public void shouldRunTwoQueriesInOneTransaction() throws SQLException {
     }
 
     @Test
-    public void shouldRunOnTransactionCompletedCallback() throws SQLException {
-        when(db.getPreparedStatement().executeBatch())
-            .thenReturn(new int[]{1, 1});
-
-        final boolean[] cbExecuted   = {false};
-        DaoObservable   daoObsWithCb = ((DaoObservable)dao.updateSuccess()).doOnTransactionCompleted(() -> cbExecuted[0] = true);
```

Comment:
I think we need to keep this, but perhaps you could store the `Action0` inside `DaoObservable` and then just run those after the transaction has succeeded? Then you do not need to subscribe any Observables to get this job done.